docker-images/keycloak/default-medco-realm.json

```diff
@@ -150,6 +150,11 @@
         "clientRole" : true,
```

Comment:
Did you test the changes made in that file to see if it is correctly into Keycloak at the first startup?

So far I don't think we made changes directly to the file ourselves, as it is produced from an export from Keycloak. Instead I was manually exporting the configuration and replacing altogether the file.